docs/lib/graphqlapi/fragments/ios/query-data.md

```diff
@@ -121,3 +119,87 @@ func listTodos() -> AnyCancellable {
 </amplify-block>
 
 </amplify-block-switcher>
+
+### List subsequent pages of items
+
+For large data sets, you'll need to paginate through the results. After receiving the first page of results, you can check if there is a subsequent page and obtain the next page.
+
+```swift
+var todos: [Todo] = []
+var currentPage: List<Todo>?
+
+func listTodos() {
```

Comment:
To make the lifecycle clearer from an app developer perspective, maybe this could be `listFirstPage` or `loadFirstPage`, and the method below could have a parallel construction `listNextPage` or `loadNextPage` or similar?

Reply:
updated to `listFirstPage` and `listNextPage` since the section is called "List Query"